.github/dependabot.yml

```diff
@@ -29,7 +29,7 @@ updates:
       windows:
         patterns:
           - "windows*"
-          - "winreg"
+          - "windows-registry"
```

Comment:
It’s best to change that in a separate, dedicated PR, because it takes a special approval that I can’t give you directly. But in this case, I believe the only required change is removing `winreg`, because `windows*` is already covering `windows-registry`.

Reply:
I reverted this change. I'll open a separate PR soon.